tests/core/component.test.js

```diff
@@ -494,7 +510,7 @@ suite('Component', function () {
       });
 
       el.hasLoaded = true;
-      el.setAttribute('dummy', '');
+      el.setAttribute('dummy', 'color: red');
```

Comment:
The value of `attrValue` is only set _once_ a value is set. Without setting any properties, `attrValue` isn't allocated (saving memory). This test inspects the `attrValue` which is internal and thus this change

Furthermore the test only cares about whether or not the selector property `el` is being cloned or not. So setting a different property to ensure `attrValue` is allocated has no bearing on the tested behaviour.